test/expected-pretty.js

```diff
@@ -0,0 +1,115 @@
+/*  */
+
+// Regular import
+import { Something } from 'some-module';
+
+// Import types
+
+
+// Typed function
+async function test(x, y /*.*/  /*.*/ , z /*.*/  /*.*/  = 123) {
+  // Typed expression
+  return await (x);
+}
+
+// Interface
+
+
+// Exported interface
+
+
+// Interface extends
+
+
+// Implements interface
+class Bar extends Other  /*.*/   {
+  // Class Property with default value
+  answer = 42;
+
+  // Class Property
+  
```

Comment:
Perhaps this pretty printer could be nicer if it removed whole lines when the only content on the line is a removed node, but this first pass was simpler.